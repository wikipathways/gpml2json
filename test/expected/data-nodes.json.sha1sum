149f5f957877b63e2d6346b2aaf68cf0400149be ?-
