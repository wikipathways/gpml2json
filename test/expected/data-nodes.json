{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["ae6ab","db4be","bbd97","c7fc2","ec253","d4b93","a9150","fca47","e78e2","adfa3","ca37f","bb1e3","eec23","cfb0e","c725d","c9952","cb8a4","ae9a4","e01b4","c0772","dd121","ad34e","f52c9","b5f5f","aa3dd","ab90e"],"drawAs":"rect","gpmlElementName":"Pathway","height":786.4815301392441,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":602.9155931563928,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20130815","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/fd34e9516dbcc0a7ab9dd86419bc88e26b38334c2a5f91c1c0ba372ab9810ea8"},"entityMap":{"bbd97":{"id":"bbd97","textContent":"CCR5","wpType":"GeneProduct","x":21.124220151396102,"y":61.794871794871796,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32768,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"dbConventionalName":"Entrez Gene","dbId":"1234","type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"ae6ab":{"id":"ae6ab","x":381.8599546201717,"y":10.740619456971984,"backgroundColor":"#333333","fontFamily":"Helvetica","fontSize":10,"height":342.74091068227204,"rotation":0,"drawAs":"rect","verticalAlign":"middle","width":105.05563853622105,"zIndex":-2,"textAlign":"center","color":"#000000","textDecoration":"none","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"c7fc2":{"id":"c7fc2","textContent":"Statin","wpType":"Metabolite","x":21.124220151396102,"y":113.0769230769231,"color":"#0000ff","fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32768,"textAlign":"center","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"dbConventionalName":"ChEBI","dbId":"35664","type":["DataNode","SingleFreeNode","Metabolite"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"db4be":{"id":"db4be","x":257.5964008832856,"y":10.740619456971956,"backgroundColor":"#333333","fontSize":10,"height":342.7409106822721,"rotation":0,"drawAs":"rect","verticalAlign":"middle","width":105.05563853622105,"zIndex":16384,"textAlign":"center","color":"#000000","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"ec253":{"id":"ec253","textContent":"Statin Pathway","wpType":"Pathway","x":19.124220151396088,"y":168.93490565622176,"color":"#14961e","fontSize":12,"fontWeight":"bold","height":20,"drawAs":"none","verticalAlign":"middle","width":98.77373903227738,"zIndex":32768,"textAlign":"center","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","borderWidth":0,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"dbConventionalName":"WikiPathways","dbId":"WP1","type":["DataNode","SingleFreeNode","Pathway"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"d4b93":{"id":"d4b93","textContent":"B8LFP0_HUMAN","wpType":"Protein","x":21.124220151396116,"y":206.63265306122446,"fontSize":10,"height":32.36734693877554,"verticalAlign":"middle","width":93.87577984860388,"zIndex":32768,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"dbConventionalName":"Uniprot-TrEMBL","dbId":"B8LFP0","type":["DataNode","SingleFreeNode","Protein"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"a9150":{"id":"a9150","textContent":"Rna","wpType":"Rna","x":21.124220151396102,"y":278.5886314988484,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32768,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","Rna"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"fca47":{"id":"fca47","textContent":"Unknown","x":21.124220151396102,"y":328.16128107149797,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32768,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"wpType":"Unknown","type":["DataNode","SingleFreeNode","Unknown"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"e78e2":{"id":"e78e2","textContent":"Unknown","x":270.1242201513961,"y":305.5800751886047,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32769,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"wpType":"Unknown","type":["DataNode","SingleFreeNode","Unknown"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"adfa3":{"id":"adfa3","textContent":"GeneProduct","wpType":"GeneProduct","x":270.1242201513961,"y":43.19315148781283,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32770,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"ca37f":{"id":"ca37f","textContent":"Pathway","wpType":"Pathway","x":270.1242201513961,"y":146.0816660551592,"color":"#14961e","fontSize":12,"fontWeight":"bold","height":20,"drawAs":"none","verticalAlign":"middle","width":80,"zIndex":32771,"textAlign":"center","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","borderWidth":0,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","Pathway"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"bb1e3":{"id":"bb1e3","textContent":"Metabolite","wpType":"Metabolite","x":270.1242201513961,"y":94.4752027698641,"color":"#0000ff","fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32772,"textAlign":"center","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","Metabolite"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"eec23":{"id":"eec23","textContent":"Rna","wpType":"Rna","x":270.1242201513961,"y":256.00742561595507,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32773,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","Rna"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"cfb0e":{"id":"cfb0e","textContent":"Protein","wpType":"Protein","x":270.1242201513961,"y":192.75956236809202,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32774,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","Protein"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"c725d":{"id":"c725d","textContent":"Unknown","x":394.38777388828225,"y":304.37768534543807,"color":"#333333","backgroundColor":"#ededed","fontFamily":"Helvetica","fontSize":10,"height":20,"drawAs":"rect","borderRadius":15,"verticalAlign":"middle","width":80,"zIndex":32776,"textAlign":"center","textDecoration":"none","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"wpType":"Unknown","type":["DataNode","SingleFreeNode","Unknown"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"c9952":{"id":"c9952","textContent":"GeneProduct","wpType":"GeneProduct","x":394.38777388828225,"y":41.99076164464623,"color":"#333333","backgroundColor":"#ededed","fontFamily":"Helvetica","fontSize":10,"height":20,"drawAs":"rect","borderRadius":15,"verticalAlign":"middle","width":80,"zIndex":32777,"textAlign":"center","textDecoration":"none","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"cb8a4":{"id":"cb8a4","textContent":"Pathway","wpType":"Pathway","x":394.38777388828225,"y":144.87927621199265,"color":"#ededed","backgroundColor":"#14961e","fontFamily":"Helvetica","fontSize":12,"fontWeight":"bold","height":20,"drawAs":"rect","borderRadius":15,"verticalAlign":"middle","width":80,"zIndex":32778,"textAlign":"center","textDecoration":"none","fontStyle":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","Pathway"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"ae9a4":{"id":"ae9a4","textContent":"Metabolite","wpType":"Metabolite","x":394.38777388828225,"y":93.27281292669755,"color":"#ededed","backgroundColor":"#0000ff","fontFamily":"Helvetica","fontSize":10,"fontWeight":"bold","height":20,"drawAs":"rect","borderRadius":15,"verticalAlign":"middle","width":80,"zIndex":32779,"textAlign":"center","textDecoration":"none","fontStyle":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","Metabolite"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"e01b4":{"id":"e01b4","textContent":"Rna","wpType":"Rna","x":394.38777388828225,"y":254.80503577278841,"color":"#333333","backgroundColor":"#ededed","fontFamily":"Helvetica","fontSize":10,"height":20,"drawAs":"rect","borderRadius":15,"verticalAlign":"middle","width":80,"zIndex":32780,"textAlign":"center","textDecoration":"none","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","Rna"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"c0772":{"id":"c0772","textContent":"Protein","wpType":"Protein","x":394.38777388828225,"y":191.55717252492536,"color":"#333333","backgroundColor":"#ededed","fontFamily":"Helvetica","fontSize":10,"height":20,"drawAs":"rect","borderRadius":15,"verticalAlign":"middle","width":80,"zIndex":32781,"textAlign":"center","textDecoration":"none","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","Protein"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"dd121":{"id":"dd121","textContent":"Unknown","x":145.38777388828225,"y":328.16128107149797,"color":"#ededed","backgroundColor":"#333333","fontFamily":"Helvetica","fontSize":10,"fontWeight":"bold","height":20,"drawAs":"rect","borderRadius":15,"verticalAlign":"middle","width":80,"zIndex":32783,"textAlign":"center","textDecoration":"none","fontStyle":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"wpType":"Unknown","type":["DataNode","SingleFreeNode","Unknown"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"ad34e":{"id":"ad34e","textContent":"GeneProduct","wpType":"GeneProduct","x":145.38777388828225,"y":61.794871794871796,"color":"#ededed","backgroundColor":"#333333","fontFamily":"Helvetica","fontSize":10,"fontWeight":"bold","height":20,"drawAs":"rect","borderRadius":15,"verticalAlign":"middle","width":80,"zIndex":32784,"textAlign":"center","textDecoration":"none","fontStyle":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"f52c9":{"id":"f52c9","textContent":"Pathway","wpType":"Pathway","x":145.38777388828225,"y":168.93490565622173,"color":"#ededed","backgroundColor":"#14961e","fontFamily":"Helvetica","fontSize":12,"fontWeight":"bold","height":20,"drawAs":"rect","borderRadius":15,"verticalAlign":"middle","width":80,"zIndex":32785,"textAlign":"center","textDecoration":"none","fontStyle":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","Pathway"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"b5f5f":{"id":"b5f5f","textContent":"Acetyl-CoA","wpType":"Metabolite","x":145.38777388828225,"y":113.0769230769231,"color":"#ededed","backgroundColor":"#0000ff","fontFamily":"Helvetica","fontSize":10,"fontWeight":"bold","height":20,"drawAs":"rect","borderRadius":15,"verticalAlign":"middle","width":80,"zIndex":32786,"textAlign":"center","textDecoration":"none","fontStyle":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"dbConventionalName":"HMDB","dbId":"HMDB01206","type":["DataNode","SingleFreeNode","Metabolite"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"aa3dd":{"id":"aa3dd","textContent":"Rna","wpType":"Rna","x":145.38777388828225,"y":278.5886314988484,"color":"#ededed","backgroundColor":"#333333","fontFamily":"Helvetica","fontSize":10,"fontWeight":"bold","height":20,"drawAs":"rect","borderRadius":15,"verticalAlign":"middle","width":80,"zIndex":32787,"textAlign":"center","textDecoration":"none","fontStyle":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","Rna"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"ab90e":{"id":"ab90e","textContent":"Protein","wpType":"Protein","x":145.38777388828225,"y":221.03066867772765,"color":"#ededed","backgroundColor":"#333333","fontFamily":"Helvetica","fontSize":10,"fontWeight":"bold","height":20,"drawAs":"rect","borderRadius":15,"verticalAlign":"middle","width":80,"zIndex":32788,"textAlign":"center","textDecoration":"none","fontStyle":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","Protein"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"}}}
